.changeset/real-planets-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@orchestrator-ui/orchestrator-ui-components': major
+---
+
+1479 csvDownload (Breaking change): Changed the parameters of initiateCsvFileDownload for directly exporting data to CSV. GroupedTable: The table has a small header with a button "Collapse / Expand", this can now be overridden. Introduced a utility type to help ensuring that all the props of an object will be of type "string". Typically useful for the csvDownload functionality
```

Comment:
this should not be a single line and you should give more context for the breaking change, so the changes can be made by people that use the package

Reply:
Yes, good one. Will do.

Reply:
Updated the file manually, but i am not sure the changeset plugin would handle this correctly. By right we should use the cli to enter the changes, which is a single line. The changelog.md file, where the changes are moved to by the changeset-plugin, seems to only have single lines.